AISKU/Tests/AISKUSize.Tests.ts

```diff
@@ -45,7 +45,7 @@ export class AISKUSizeCheck extends TestClass {
                         return;
                     } else {
                         return response.text().then(text => {
-                            let size = Math.ceil(pako.deflate(text).length/1024);
+                            let size = Math.ceil((pako.deflate(text).length/1024) * 100) / 100.0;
```

Comment:
Just updating test so it displays how close we are to the max value -- to 2 decimal places